content/md/assets/upload-assets_fr

```diff
@@ -0,0 +1,46 @@
+---
+id: upload-assets
+themes: manage-your-assets, import-export-data
+title: "**Upload** assets"
```

Comment:
Translate the title also. ;)

Reply:
J'ai traduit le titre mais est-ce qu'il faut aussi traduire tout le reste ?  :)
id: _upload-assets_
themes: _manage-your-assets, import-export-data_
title: "**Charger** des ressources numériques"
popular: false
ee-only: true
related: _work-with-assets, assets-transformation_